src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -111,7 +117,11 @@ PARAM_UROW_IN("labels", "Labels for the training set.", "l");
 
 // Classification options.
 PARAM_MATRIX_IN("test", "Test dataset.", "T");
+/* 
+* The PARAM_UROW_OUT("output") is deprecated and can be removed in mlpack4.0.0.
```

Comment:
Should we adjust the wording here, to 'will be removed in mlpack 4.0.0.'?

Reply:
Made changes